dist/wavesurfer.js

```diff
@@ -1192,14 +1192,15 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
      *  @param  {String}        url         path to media file
      *  @param  {HTMLElement}   container   HTML element
      *  @param  {Array}         peaks       array of peak data
+	 *  @param  {String}        preload     HTML 5 preload attribute value
```

Comment:
Arf, GitHub web editor is really not friendly with indentation display... :/
Do you want me to fix this ou can you do it ?